functions-python/extract_bb/function_config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "extract-bounding-box",
+  "description": "Extracts the bounding box from a dataset",
+  "entry_point": "extract_bounding_box",
+  "timeout": 540,
+  "memory": "2Gi",
+  "trigger_http": false,
+  "include_folders": ["database_gen", "helpers"],
+  "environment_variables": [],
+  "secret_environment_variables": [
+    {
+      "key": "FEEDS_DATABASE_URL"
+    }
+  ],
+  "ingress_settings": "ALLOW_INTERNAL_AND_GCLB",
+  "max_instance_request_concurrency": 20,
```

Comment:
Yes that's a good point. In case multiple large feeds within the same instance, we might run into memory issues. What do you think about having similar configuration as the [`batch-process-dataset`](https://github.com/MobilityData/mobility-feed-api/blob/main/functions-python/batch_process_dataset/function_config.json)?

Reply:
That's a good idea, batch-process-dataset should have a similar memory footprint.